test.js

```diff
@@ -156,6 +156,48 @@ test('`fullPage` option', async t => {
 	t.true(size.height > 100);
 });
 
+test('`fullPage` option - lazyloading', async t => {
+	const server = await createTestServer();
+
+	server.get('/', async (request, response) => {
+		response.end(`
+			<body>
+				<div style="display: grid; grid-template-columns: repeat( auto-fill, minmax(150px, 1fr) );" id="grid"></div>
+				<script>
+					const numItemsToGenerate = 250; //how many gallery items you want on the screen
+					const numImagesAvailable = 1100; //how many total images are in the collection you are pulling from
+					const imageWidth = 150; //your desired image width in pixels
+					const imageHeight = 150; //desired image height in pixels
+					const collectionID = 9717149; //the collection ID from the original url
+					function renderGalleryItem(randomNumber){
+					fetch('https://source.unsplash.com/collection/'+collectionID+'/'+imageWidth+'x'+imageHeight+'/?sig='+randomNumber)
+					.then((response)=> {
+						let galleryItem = document.createElement('div');
+						galleryItem.classList.add('gallery-item');
+						galleryItem.innerHTML = '<img class="gallery-image" src="'+response.url+'" alt="gallery image" loading="lazy" />';
+						document.getElementById('grid').appendChild(galleryItem);
+					})
+					}
+					for(let i=0;i<numItemsToGenerate;i++){
+					let randomImageIndex = Math.floor(Math.random() * numImagesAvailable);
+					renderGalleryItem(randomImageIndex);
+					}
```

Comment:
Can you clean up the code style here? It's a bit messy. Use async/await, template literals, etc.

Reply:
Also move the string to the top-level.

Reply:
Gotcha. Will learn a few things here and there and proceed as such. Thanks for the feedback